podcasts/UpNextViewController.swift

```diff
@@ -66,7 +68,9 @@ class UpNextViewController: UIViewController, UIGestureRecognizerDelegate {
             upNextTable.register(UINib(nibName: "NothingUpNextCell", bundle: nil), forCellReuseIdentifier: UpNextViewController.noUpNextCell)
             upNextTable.register(UINib(nibName: "UpNextNowPlayingCell", bundle: nil), forCellReuseIdentifier: UpNextViewController.nowPlayingCell)
             upNextTable.backgroundView = nil
-
+            if showingInTab {
+                upNextTable.applyInsetForMiniPlayer()
```

Comment:
I think we should move this to `viewDidAppear`.

If a user has an empty Up Next and opens the app, then fills it with a lot of items, the last one will be hidden till the app is restarted. If we move to `viewDidAppear` it should fix this.

Reply:
Good call I moved this to viewDidAppear as you suggested.